doc/CMakeLists.txt

```diff
@@ -380,7 +388,10 @@ endif()
 #
 # Dependencies and final targets
 #
-add_dependencies(html content kconfig devicetree)
+
+# nRF-specific: No 'kconfig' dependency here, because we use a shared Kconfig
+# documentation set instead
+add_dependencies(html content doxy_real_modified_times devicetree)
```

Comment:
`doxy_real_modified_times` target does not exist anymore.

Reply:
removed